sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -532,9 +532,13 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
       case logical.Intersect(left, right) =>
         throw new IllegalStateException(
           "logical intersect operator should have been replaced by semi-join in the optimizer")
-      case logical.Except(left, right) =>
+      case logical.Except(left, right, false) =>
         throw new IllegalStateException(
           "logical except operator should have been replaced by anti-join in the optimizer")
+      case logical.Except(left, right, true) =>
+        throw new IllegalStateException(
+          "logical except operator should have been replaced by union, aggregate" +
```

Comment:
except all

Reply:
there is no operator called "except all " ?

Reply:
hmm, it looks no diff to above one. Maybe `except (all) operator`?

Reply:
ok.. will change to except (all).. the difference was in the replaced operators, actually.